scene/resources/default_theme/default_theme.cpp

```diff
@@ -1011,6 +1011,10 @@ void fill_default_theme(Ref<Theme> &theme, const Ref<Font> &default_font, Ref<Te
 	theme->set_icon("resizer", "GraphEditMinimap", icons["resizer_nw"]);
 	theme->set_color("resizer_color", "GraphEditMinimap", Color(1, 1, 1, 0.85));
 
+	// EditorSpinSlider
+	theme->set_color("label_color", "EditorSpinSlider", control_font_color);
+	theme->set_color("read_only_label_color", "EditorSpinSlider", control_font_disabled_color);
+
```

Comment:
Adding the new theme properties to the default theme is the only way it would allow me to add documentation for them. However, none of the other editor-only nodes have documentation so I was unsure of whether it is better to add it or not.

If it's not desired to have documentation for the theme properties of editor-only nodes, I can remove it :)

Reply:
Nevermind, I forgot about the Editor Theme tab in the theme editor, which is where you can see the theme properties for other editor-only nodes, like EditorProperty.